server/api/store.go

```diff
@@ -12,4 +12,10 @@ type Store interface {
 	ListApps() []*apps.App
```

Comment:
An improvement would be to use the DAL pattern used on the server and have an `AppsStore` wrapper and a `ManifestStore` wrapper and then rename StoreApp/StoreManifest to Save and `LoadApp/LoadManifest` to `Get` and 'ListApps` to `GetAll`

Reply:
Whatever pattern we use here, I'd prefer that it is consistent across all stores in `plugin-apps`. `GetAll` seems more intention-revealing than `ListApps`.